fsw/src/ci_lab_app.c

```diff
@@ -47,26 +47,19 @@
  */
 typedef union
 {
-    CFE_MSG_Message_t MsgHdr;
+    CFE_MSG_Message_t Msg;
     uint8             bytes[CI_LAB_MAX_INGEST];
     uint16            hwords[2];
 } CI_LAB_IngestBuffer_t;
```

Comment:
The "hwords" member here really should not be needed.  AFAIK it served two purposes - to align the buffer to 16 bits (not needed if using CFE_MSG_Message_t) and in the event the message didn't validate it would actually report the value of the first two words in an error message.  But that message is fundamentally flawed because if the length check doesn't work the values probably are undefined/uninitialized and thereby meaningless.